src/start/PasswordAuthView.js

```diff
@@ -43,7 +45,9 @@ class PasswordAuthView extends React.Component {
       progress: false,
       email: props.email || config.defaultLoginEmail,
       password: config.defaultLoginPassword,
-      error: ''
+      error: '',
+      ishidden: true,
```

Comment:
Letter casing should be 'isHidden'. Also it should probably be more descriptive, like 'isPasswordHidden'.